MidasCivil_Adapter/Convert/ToMidasCivil/Properties/FromMetaData.cs

```diff
@@ -0,0 +1,92 @@
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Text;
+using BH.oM.Adapters.MidasCivil;
+using System.Collections.Generic;
+
+namespace BH.Adapter.Adapters.MidasCivil
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static StringBuilder FromMetaData(MetaData midasMetaData)
+        {
+            StringBuilder midasMetaDataSB = new StringBuilder();
+            List<Review> reviews = midasMetaData.Reviews;
+            midasMetaDataSB.AppendLine("*PROJINFO");
+            midasMetaDataSB.AppendLine($"PROJECT={midasMetaData.ProjectNumber}");
+            midasMetaDataSB.AppendLine($";DESIGNSTAGE={midasMetaData.DesignStage}");
+            midasMetaDataSB.AppendLine($";PROJECTLEAD={midasMetaData.ProjectLead}");
+            midasMetaDataSB.AppendLine($"REVISION={midasMetaData.Revision}");
+            midasMetaDataSB.AppendLine($"USER={midasMetaData.Author}");
+            midasMetaDataSB.AppendLine($"EMAIL={midasMetaData.Email}");
+            midasMetaDataSB.AppendLine($"ADDRESS={midasMetaData.Location.Replace(System.Environment.NewLine, ", ")}");
+            midasMetaDataSB.AppendLine($"CLIENT={midasMetaData.Client}");
+            midasMetaDataSB.AppendLine($"TITLE={midasMetaData.ProjectName}");
+            midasMetaDataSB.AppendLine($"ENGINEER={midasMetaData.Author}");
+            midasMetaDataSB.AppendLine($"EDATE={midasMetaData.CreationDate.ToString("yyyy-MM-dd")}");
+            midasMetaDataSB.AppendLine($";DESCRIPTION={midasMetaData.Description}");
+            midasMetaDataSB.AppendLine($";DISCIPLINE={midasMetaData.Discipline}");
+
+            if(reviews.Count > 4)
+            {
+                reviews.RemoveRange(1, reviews.Count - 2);
+                Engine.Reflection.Compute.RecordWarning("The number of reviews exceeds the limit of MidasCivil. Only the first, last and second to last reviews will be recorded.");
+            }
+            if(reviews.Count >= 1)
+            {
+                midasMetaDataSB.AppendLine($"CHECK1={reviews[0].Reviewer}");
+                midasMetaDataSB.AppendLine($"CDATE1={reviews[0].ReviewDate.ToString("yyyy-MM-dd")}");
```

Comment:
We should also put the reviewers comments in with `;` so that when it's pulled back that information is not lost.

Reply:
Let's leave this one within the Midas functionality for now, and potentially expand on this afterwards.